src/groovy/org/pih/warehouse/api/StockMovementItem.groovy

```diff
@@ -102,6 +102,7 @@ class StockMovementItem {
     }
 
     Map toJson() {
+        ShipmentItem shipmentItem = ShipmentItem.get(id);
```

Comment:
We should pass the order number into the stock movement item when it's created. Look to see how the orderItemId is set and do the same.

Reply:
@jmiranda I have changed this one.